opentracing-cdi-example/src/main/java/io/opentracing/contrib/cdi/example/OrderService.java

```diff
@@ -0,0 +1,29 @@
+package io.opentracing.contrib.cdi.example;
+
+import io.opentracing.SpanContext;
+import io.opentracing.contrib.cdi.Traced;
+
+import javax.inject.Inject;
+import java.util.logging.Logger;
+
+/**
+ * This is a regular synchronous stateless EJB. It demonstrates how to get the span context for the span it's wrapped
+ * on. This can be used to pass down the call chain, create child spans or add baggage items.
+ *
+ * @author Juraci Paixão Kröhling
+ */
+@Traced
+public class OrderService {
+    private static final Logger log = Logger.getLogger(OrderService.class.getName());
+
+    @Inject
+    InventoryService inventoryService;
+
+    @Inject
+    SpanContext spanContext;
```

Comment:
Note that this has a different semantic as if this had been a method parameter. For a method parameter, this is a "child" span of the caller:

```java
public void action() {
  otherService.anotherAction(this.spanContext);
}
```

The interceptor will replace the span context received by `anotherAction` by a child span of the span sent by `action`.

When injecting the `SpanContext`, we get it from the request attributes, so, it would be misleading to create this span as a direct child of the request. I think we should better let the application decide, but I'm open for suggestions.